templates/checkout/form-confirm-payment.php

```diff
@@ -89,17 +89,17 @@
 
 			<section class="llms-checkout-section">
 
-				<h4 class="llms-form-heading"><?php _e( 'Payment Details', 'lifterlms' ); ?></h4>
+				<h4 class="llms-form-heading"><?php esc_html_e( 'Payment Details', 'lifterlms' ); ?></h4>
 				<div class="llms-checkout-section-content llms-form-fields">
 
 					<div class="llms-payment-method">
 						<?php do_action( 'lifterlms_checkout_confirm_before_payment_method', $gateway_id ); ?>
-						<span class="llms-gateway-title"><span class="llms-label"><?php _e( 'Payment Method:', 'lifterlms' ); ?></span> <?php echo $selected_gateway->get_title(); ?></span>
+						<span class="llms-gateway-title"><span class="llms-label"><?php esc_html_e( 'Payment Method:', 'lifterlms' ); ?></span> <?php echo esc_html( $selected_gateway->get_title() ); ?></span>
 						<?php if ( $selected_gateway->get_icon() ) : ?>
-							<span class="llms-gateway-icon"><?php echo $selected_gateway->get_icon(); ?></span>
+							<span class="llms-gateway-icon"><?php echo wp_kses_post( $selected_gateway->get_icon() ); ?></span>
```

Comment:
These changes are breaking some of the checkout functionality. The icons beside gateways don't appear (ie. credit card images, "powered by stripe" for credit card, and the first gateway isn't checked by default (but is bolded)